docs/src/pages/reference/configuration-reference.md

```diff
@@ -3,22 +3,44 @@ layout: ~/layouts/MainLayout.astro
 title: Configuration Reference
 ---
 
-To configure Astro, add an `astro.config.mjs` file in the root of your project. All settings are optional.
-
-You can view the full configuration API (including information about default configuration) on [GitHub.](https://github.com/withastro/astro/blob/latest/packages/astro/src/%40types/astro.ts)
+To configure Astro, add an `astro.config.mjs` file to the root of your project.
 
 ```js
-// Example: astro.config.mjs
-
-// @type-check enabled!
-// VSCode and other TypeScript-enabled text editors will provide auto-completion,
-// helpful tooltips, and warnings if your exported object is invalid.
-// You can disable this by removing "@ts-check" and `@type` comments below.
-
-// @ts-check
-export default /** @type {import('astro').AstroUserConfig} */ (
-  {
-    // ...
-  }
-);
+export default /** @type {import('astro').AstroUserConfig} */ ({
+  projectRoot: './',
+  public: './public/',
+  dist: './dist/',
+  src: './src/',
+  pages: './pages/',
+  renderers: [
+    '@astrojs/renderer-svelte',
```

Comment:
I think the renderers should be commented out, that way should they need them they merely need to uncomment them to turn them on,

Reply:
These represent the default values. I’ve added a note about this at the top of the object. <sup>[*](https://github.com/withastro/astro/pull/2239/files#diff-04d7d285adad1eb8d5e86bf573dd5a87657acd0ebfd9b0c30c77e9de40aa820bR10)</sup>